### PR TITLE
Logs: remove duplicated

### PR DESCRIPTION
**Pull request by gpotter2**
PR:
- remove `onlyOnce` param: dirty implementation (was mine at some point...) + useless with latest versions
- Move PYX and MATPLOTLIB detection to an external file because when loading:
   - config is imported
   - data is imported from config
   - consts is imported from data
   - warnings are triggered before `interact()` setups the logs.

We have moved those functions several time, from arch to consts...
I'm finally dropping them in a dedicated file so they don't interfere with any other part of scapy.

### CODE REVIEW

File: scapy/config.py [hunk 1]
```diff
@@ -462,10 +461,6 @@ class Conf(ConfClass):
     for m in ["inet6","dhcp6"]:
         if m in Conf.load_layers:
             Conf.load_layers.remove(m)
-    
-if not Conf.crypto_valid:
-    log_scapy.warning("Crypto-related methods disabled for IPsec, Dot11 "
-                      "and TLS layers (needs python-cryptography v1.7+).")
```
Comment (gpotter2) [inline scapy/config.py]: The warning is already triggered in each of those files

File: scapy/plist.py [hunk 2]
```diff
@@ -17,10 +17,10 @@
 from scapy.base_classes import BasePacket,BasePacketList
 from scapy.utils import do_graph,hexdump,make_table,make_lined_table,make_tex_table,get_temp_file
 
-from scapy.consts import plt, MATPLOTLIB_INLINED, MATPLOTLIB_DEFAULT_PLOT_KARGS
+from scapy.extlib import plt, MATPLOTLIB_INLINED, MATPLOTLIB_DEFAULT_PLOT_KARGS
 from functools import reduce
 import scapy.modules.six as six
-from scapy.modules.six.moves import filter, range, zip
+from scapy.modules.six.moves import range
```
Comment (p-l-) [inline scapy/plist.py]: why did you remove `zip`? It is used!

Comment (gpotter2) [reply scapy/plist.py]: Right. Updated

**Comment by guedou** [comment]
@p-l- are you OK with these changes ?
